README.md

```diff
@@ -11,4 +13,12 @@ then
 
 ---
 
+## Available Endpoints:
+
+### /api
+
+- responds with a list of all available endpoints
```

Comment:
Don't forget to update the endpoints json as well as you make further endpoints!